src/compiler/raw-file.js

```diff
@@ -11,8 +12,16 @@ export default class RawFileCompiler {
         test.fixture = fixture;
 
         test.fn = async testRun => {
-            for (var i = 0; i < test.commands.length; i++)
-                await testRun.executeCommand(test.commands[i]);
+            for (var i = 0; i < test.commands.length; i++) {
+                var command  = cloneDeep(test.commands[i]);
```

Comment:
It better be done with object passed to ctor, like we did for options. Command class should use options class internally and call it's ctor (to trigger validation). Also, it should validate selector.